.travis.yml

```diff
@@ -5,25 +5,32 @@ arch:
   - arm64
 
 go:
-  - 1.8
-  - 1.9
-  - 1.10.x
-  - 1.11.x
-  - 1.12.x
   - 1.13.x
   - 1.14.x
+  - 1.15.x
 
```

Comment:
As a test is run for go 1.12.x, it is OK to remove `-1.12.x` from here?

Reply:
Yes, it's add a new test case. You can see it ran here: https://github.com/DataDog/datadog-go/pull/165/checks?check_run_id=986515245